seed/challenges/03-front-end-libraries/front-end-libraries-projects.json

```diff
@@ -121,7 +117,6 @@
       "image": "u5RTBt4",
       "tests": [],
       "type": "zipline",
-      "isRequired": true,
```

Comment:
The issue calls for removing the logic but does not say anything about removing this information. I'd suggest we leave it in place for now

Reply:
I had restore all properties in json files to original state.